nodejs/sqlcommenter-nodejs/packages/sqlcommenter-sequelize/index.js

```diff
@@ -12,29 +12,44 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-const {hasComment, toW3CTraceContext} = require('./util');
-const {tracer} = require('@opencensus/nodejs');
+const {hasComment} = require('./util');
+const {OpenCensusProvider, OpenTelemetryProvider} = require('./provider')
 
 const defaultFields = {
     'route': true,
     'tracestate': false,
     'traceparent': false,
 };
 
+const providerMap = {
+    'OpenCensus': new OpenCensusProvider(),
+    'OpenTelemetry': new OpenTelemetryProvider(),
+}
+
 /**
  * All available variables for the commenter are on the `util.fields` object
  * passing the excludes parameter will result in each item being excluded from
  * the commenter output
  * 
  * @param {Object} sequelize 
- * @param {Object} includes a map of values to be optionally included.
+ * @param {Object} include - A map of values to be optionally included.
+ * @param {Object} options - A configuration object specifying where to collect trace data from. Accepted fields are:
+ *  TraceProvider: Should be either 'OpenCensus' or 'OpenTelemetry', indicating which library to collect trace data from.
  * @return {void}
  */
-exports.wrapSequelize = (sequelize, include={}) => {
+exports.wrapSequelize = (sequelize, include={}, options={TraceProvider: 'OpenCensus'}) => {
```

Comment:
Why 'OpenCensus' and why not 'opencensus' and 'opentelemetry'? CamelCase for a string option might be too tough on the eyes for users who can type 'Opencensus' or 'Opentelemetry' accidentally.

Reply:
I think I'll make the config accept any capitalization of the name to simplify things.